### DISCUSSION TITLE
Laravel backpack overwrite update blade file for just one CRUD

### DISCUSSION THREAD

**Discussion by blessen131** [Laravel-Backpack/community-forum#1009] [General] [answered]
1. I am trying to hide 'edit' button from a trashed item just one CRUD, Copied update blade file into 
resources/views/vendor/backpack/crud/buttons/ and updated the first line

@if ($crud->hasAccess('update', $entry) && !$entry->trashed())
Is it possible to load the update blade with conditon(!$entry->trashed()) just one controller?

I cannot find `setAccessCondition` on backpack:5.x. None of the other models has 'deleted_at' column.

2. When I try to "show" an entry using Backpack ShowOperation, and that entry has been soft-deleted, the application throws a 404 not found. I tried updating 'softDeletes' value in `/src/config/backpack/operations/show.php`, getting a blank page instead of 404!

Is it possible to display the soft-deleted entries in the show operation ?

laravel: 8.x Backpack: 5.x

**Comment by pxpm** [comment, marked as answer]
@blessen131 I think I answered you (or someone with a similar question) on stackoverflow. 

I am afraid in 5.x we didn't had `setAccessCondition()` that would allow us to change the visibility without having to overwrite the views. 

In that case you are in the right direction:
- Create a copy of the button and customize it (call it `update_trashed` or something different than the original `update` name so it does not overwrite the original button in other Controllers). 
- You will need to modify the button that the UpdateOperation adds:
```php
protected function setupListOperation()
{
    // ... your columns, filters etc

    CRUD::modifyButton('update', ['content' => 'crud::buttons.update_trashed']); 
    // you can place the new button wherever you want, as long as you provide the proper view path here. 
}
```
About the `ShowOperation` maybe you are in a version that has some bug ? I've just tested on the latest version and it works, so I dunno how to help you.

We are not providing bug fixes for v5 for a long time, so if it's actually a bug a not some problem with your implementation, i'd suggest you take the same approach, publish the backpack view to your resources folder, fix it and use it in your project. 

A blank page is weird, maybe you have some errors in `storage/logs` that could help to see where the issue is. 

Cheers
